tortoise/contrib/test/__init__.py

```diff
@@ -178,7 +178,9 @@ def _init_loop(self) -> None:
     async def _tearDownDB(self) -> None:
         pass
 
-    async def _setUp(self) -> None:
+    def _setUp(self) -> None:
```

Comment:
er... this method does `await` at line 190/192
Also, the loop should be initialised BEFORE the tests are run, so in the `run()` method at line 241

This should be a syntax error?